test/jsunit/block/block.test.js

```diff
@@ -75,7 +75,7 @@ describe('Filesystem Block tests', () => {
 describe('WebView Block tests', () => {
 
   beforeAll(async () => {
-    await page.goto(`${SERVER}`, { waitUntil: 'domcontentloaded' });
+    await page.goto(`${SERVER}`, { waitUntil: 'networkidle0' });
```

Comment:
need to replace 'domcontentloaded' with 'networkidle0' because the tests need to wait until the language is loaded.